src/components/KCard/KCard.vue

```diff
@@ -1,244 +1,122 @@
 <template>
-  <section
-    :aria-describedby="contentId || undefined"
-    :aria-label="title ? title : undefined"
-    :aria-labelledby="!title && slots.title ? titleId : undefined"
-    class="kong-card"
-    :class="[borderVariant, {'hover': hasHover, 'kcard-shadow': hasShadow }]"
-  >
+  <section class="k-card">
     <div
-      v-if="showCardHead"
-      class="k-card-header"
-      :class="{ 'has-status': status || slots.statusHat }"
+      v-if="$slots.title || title || $slots.actions"
+      class="card-header"
     >
       <div
-        v-if="status || slots.statusHat"
-        class="k-card-status-hat"
-      >
-        <!-- @slot Use this slot to pass status text above title -->
-        <slot name="statusHat">
-          {{ status }}
-        </slot>
-      </div>
-
-      <div
-        v-if="showCardTitleWithoutStatus"
-        :id="title ? undefined : titleId"
-        class="k-card-title"
+        v-if="$slots.title || title"
+        class="card-title"
       >
         <h4>
-          <!-- @slot Use this slot to pass title content -->
           <slot name="title">
             {{ title }}
           </slot>
         </h4>
       </div>
-
-      <div class="k-card-actions">
-        <!-- @slot Use this slot to pass actions to right side of header -->
+      <div
+        v-if="$slots.actions"
+        class="card-actions"
+      >
         <slot name="actions" />
       </div>
     </div>
-
     <div
-      v-if="showCardTitleWithStatus"
-      :id="title ? undefined : titleId"
-      class="k-card-title"
+      v-if="$slots.default || $slots.body || content || body"
+      class="card-content"
     >
-      <h4>
-        <!-- @slot Use this slot to pass title content -->
-        <slot name="title">
-          {{ title }}
-        </slot>
-      </h4>
+      <slot :name="contentSlotName">
```

Comment:
instead of this conditional, can you just add both slots and then we remove one later? 

You can add v-if logic to only show one

Reply:
Done